### PR TITLE
[BRANCH-0.4.x] Release 0.4.3

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR targets to release 0.4.3.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
   reference to a dynamically defined class from the `__main__` module.
   ([issue #131]( https://github.com/cloudpipe/cloudpickle/issues/131)).
 
+- Fixed a crash in Python 2 when serializing non-hashable instancemethods of built-in
```
Comment (HyukjinKwon) [inline CHANGES.md]: This was backported in https://github.com/cloudpipe/cloudpickle/pull/154

**Comment by HyukjinKwon** [comment]
It's late in my local time now .. I would appreciate if anyone push a tag to GitHub and make a release to PyPi ... otherwise, I will do that tomorrow. Either way works!

**Comment by rgbkrk** [comment]
Thanks @HyukjinKwon, feel free to make that tag and release when ready.

**Comment by HyukjinKwon** [comment]
I have pushed the tag and release v0.4.3 - https://github.com/cloudpipe/cloudpickle/releases/tag/v0.4.3

and made an upload to PyPi - https://pypi.python.org/pypi?:action=display&name=cloudpickle&version=0.4.3

**Comment by rgbkrk** [comment]
Excellent!

**Pull request merged**